src/main/scala/com/cloudant/clouseau/IndexService.scala

```diff
@@ -117,6 +121,11 @@ class IndexService(ctx: ServiceContext[IndexServiceArgs]) extends Service(ctx) w
         pendingSeq = newSeq
         logger.debug("Pending sequence is now %d".format(newSeq))
         'ok
+      case SetPurgeSeqMsg(newSeq: Long) =>
+        pendingPurgeSeq = newSeq
+        logger.debug("Pending purge sequence is now %d".format(newSeq))
+        commit(pendingSeq, pendingPurgeSeq)
```

Comment:
Hi @rnewson .
If we commit purgeSeq asynchronously and also the users purge a lot, the `dreyfus` will retry to delete the index again in `clouseau`.
For example, if `dreyfus` gets a purgeSeq(6) from couchdb and gets an IdxPurgeSeq(5) from `clouseau`, it will purge index from 5 and set purgeSeq to 6.
If `clouseau` is crashed before committing, `dreyfus` will get purgeSeq(5) again. And `dreyfus` will purge index from 5 again.
But I am not sure which is more expensive, delete index or commit?